### PR TITLE
Create 10.0.9 change log

### PR DESCRIPTION
**Pull request by JohnRusk**
Here's an initial draft of the change log for the new version. I prepared it by going through the commit history, back to 10.0.8.  I deliberately left out about three very minor changes.

Would be good if at least one of you guys can check that I have not missed anything important.

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -1,22 +1,57 @@
...
+1. For creating MD5 hashes when uploading, version 10.x now has the OPPOSITE default to version
+   AzCopy 8.x. Specifically, as of version 10.0.9, MD5 hashes are NOT created by default. To create
+   Content-MD5 hashs when uploading, you must now specify `--put-md5` on the command line.
+
+### New features
+
+1. Can migrate data directly from Amazon Web Services (AWS). In this high-performance data path
+   the data is read directly from AWS by the Azure Storage service. It does not need to pass through
+   the machine running AzCopy. The copy happens syncronously, so you can see it's exact progress.  
+1. Can migrate data directly from Azure Files or Azure Blobs (any blob type) to Azure Blobs (any
```
Comment (jiacfan) [inline ChangeLog.md]: >1 [](start = 0, length = 1)

What will this 1s. been shown in github page? Are they by design?

**Comment by seguler** [comment]
:shipit:

File: ChangeLog.md [hunk 2]
```diff
@@ -1,22 +1,57 @@
...
+
+1. For creating MD5 hashes when uploading, version 10.x now has the OPPOSITE default to version
+   AzCopy 8.x. Specifically, as of version 10.0.9, MD5 hashes are NOT created by default. To create
+   Content-MD5 hashs when uploading, you must now specify `--put-md5` on the command line.
+
+### New features
+
+1. Can migrate data directly from Amazon Web Services (AWS). In this high-performance data path
+   the data is read directly from AWS by the Azure Storage service. It does not need to pass through
+   the machine running AzCopy. The copy happens syncronously, so you can see it's exact progress.
```
Comment (zezha-msft) [inline ChangeLog.md]: Minor typo "its" exact progress

File: ChangeLog.md [hunk 3]
```diff
@@ -1,22 +1,57 @@
...
+
+### New features
+
+1. Can migrate data directly from Amazon Web Services (AWS). In this high-performance data path
+   the data is read directly from AWS by the Azure Storage service. It does not need to pass through
+   the machine running AzCopy. The copy happens syncronously, so you can see it's exact progress.  
+1. Can migrate data directly from Azure Files or Azure Blobs (any blob type) to Azure Blobs (any
+   blob type). In this high-performance data path the data is read directly from the source by the
+   Azure Storage service. It does not need to pass through the machine running AzCopy. The copy
+   happens syncronously, so you can see it's exact progress.
```
Comment (zezha-msft) [inline ChangeLog.md]: same minor typo, "its" exact progress

**Comment by zezha-msft** [comment]
:shipit:

File: ChangeLog.md (hunk 1 above)
Comment (JohnRusk) [reply ChangeLog.md]: Which 1s?  Do you mean the 1. in the numbered list in the doc - yes, those get auto adjusted to correct value when the markdown is displayed.
Or do you mean something else. I see "1 [](start = 0, length = 1)" in your comment, and I don't recognize that text.

**Pull request merged**